randovania/games/super_metroid/json_data/Crateria.txt

```diff
@@ -140,11 +152,14 @@ Extra - asset_id: 3
   * Normal Door to West Ocean/Door to Moat
   > Door to Crateria Kihunter Room
       Any of the following:
-          # Via top path - TODO: check for shinespark without entrance rando?
-          Grapple Beam or Space Jump
+          Any of the following:
+              # Via top path
+              Grapple Beam or Space Jump or Movement (Intermediate)
+              # 2 tap short charge from West Ocean
+              Speed Booster and Shinespark (Beginner) and Short Charge (Intermediate) and Shinespark Damage ≥ 43 and Disabled Entrance Randomizer
           Any of the following:
               # Via bottom path
-              Can Lay Bombs
+              Enabled Moat Tweaks or Can Lay Bombs
```

Comment:
Moat tweaks still needs morph, no?
(Also, didnt I do a PR for this already...?)

Reply:
If it is the moat tweaks I'm thinking of, then no. I'll check.

Reply:
These moat tweaks do need morph. (Sorry, some rando I've played recently added 2 shot blocks there so Samus can walk out after failing a CWJ.